### ISSUE TITLE
'double precision' shows as 'real' in DDL

### ISSUE THREAD

**Issue by markamccann** [jakob/Postico#709]
Table fields of type 'double precision' correctly show as 'double precision' in the Structure view under TYPE, but in the DDL view, they show up instead as 'real'.  According to [PostreSQL documentation](https://www.postgresql.org/docs/12/datatype-numeric.html), these are not the same types.  The actual command I used to create the table specified 'double precision'.

## What software versions are you using?

Postico version:  
1.5.10

macOS version:  
10.15.4

PostgreSQL version:  
12.1

**Comment by martinjankoehler** [comment]
Hello @markamccann ,

thanks for your report!

I was not able yet to reproduce this behaviour with PG12 and Postico 1.5.10.

What I did was execute:
```sql
CREATE TABLE double_precision_ddl_bug(id int, d double precision, r real);
```
Then I got this DDL, which seems OK:
![Bildschirmfoto 2020-11-05 um 16 29 02](https://user-images.githubusercontent.com/29398377/98260802-054cde80-1f84-11eb-835d-0982306baff0.jpg)

Any chance to get your command to create the table?

Best regards and thanks,
Martin
